app/src/main/java/org/hackillinois/android/database/entity/Project.kt

```diff
@@ -59,32 +47,44 @@ data class Project(
             return NO_RESOURCE
         }
 
-        var buildingKey = split[0]
-        if (split.size >= 2) {
+        val buildingName = split[0]
+
+        return if (buildingName == "Siebel") {
```

Comment:
Why don't we have DCL and Kenney anymore?

Reply:
Wrote it in a different comment, but why do we need them for mentors? Will they ever have a room in either building?

Reply:
Ohhh true. I feel dumb now. Approving.